Libplanet.Explorer/GraphTypes/BlockType.cs

```diff
@@ -13,45 +13,78 @@ public class BlockType<T> : ObjectGraphType<Block<T>>
     {
         public BlockType()
         {
+            #pragma warning disable SA1118
```

Comment:
Is it necessary? if that. IMO we need more detailed comments about [SA1118](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1118.md).

Reply:
@riemannulus 
You mean we need some justification message under that pragma disable?
Is commenting out the reason `//like this` would be enough?